ui/app/src/components/AuditGridFilterPopover/styles.ts

```diff
@@ -18,26 +18,18 @@ import { makeStyles } from 'tss-react/mui';
 
 const useStyles = makeStyles()((theme) => ({
   popover: {
-    padding: '20px',
+    padding: theme.spacing(1),
     minWidth: '300px',
     overflow: 'initial'
   },
-  popoverForm: {},
-  popoverCloseIcon: {
-    background: theme.palette.divider,
-    color: theme.palette.text.secondary,
-    position: 'absolute',
-    top: '-20px',
-    right: '-20px',
-    cursor: 'pointer',
-    '&:hover': {
-      opacity: '0.9'
+  timestampFiltersContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: theme.spacing(2),
+    '& .MuiTextField-root:first-child': {
```

Comment:
There seems to be an issue in the `DateTime/date/time` pickers and the className doesn't apply (neither under PaperProps, or classes)

Reply:
Use the field classes export

Reply:
https://mui.com/x/react-date-pickers/getting-started/#MaterialUIPickers.js